### ISSUE TITLE
Bug: Navigation Banner transparent bottom border missing when user changes colours

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#4291]
**Describe the bug**
When I view the site in Firefox with colour preferences changed and with Windows High Contrast mode I should be able to see where the banner ends, this should be represented with a line, like the start of the footer (see screen shot). This should be the same on all page types of the site. Note some page types appear to have a 2px border at the top of the navigation in the banner (see screen shot), this should only be 1px, this should also be addressed.

Note: Background colours aren’t implemented when you change colours and therefore you won’t see any background colours, hence the need for transparent border - this picks up the colours you choose

**Screenshots**
Missing border at bottom of banner
<img width="1626" alt="Screenshot 2021-01-26 at 11 41 44" src="https://user-images.githubusercontent.com/1061401/105841168-1bd76480-5fcc-11eb-96d8-96d4d58c17f8.png">

Footer example
<img width="1626" alt="Screenshot 2021-01-26 at 11 41 39" src="https://user-images.githubusercontent.com/1061401/105841139-14b05680-5fcc-11eb-8a96-8458a50ff107.png">

2px border at the top of the navigation on some pages - this should be 1px
<img width="1626" alt="Screenshot 2021-01-26 at 11 51 45" src="https://user-images.githubusercontent.com/1061401/105841794-1f1f2000-5fcd-11eb-9251-ac142c71a608.png">

**Testing notes**
- [ ] View the banner with the colour preferences changed in FireFox. Go to the ‘Preferences’ under the ‘Firefox’ menu, then under ‘Language and Appearance’, select ‘Colours’ and change the colours for text, background, unvisited and visited links, then select ‘Always’ for ‘Override the colours specified by the page with your selections above’. After testing, you will need to restore the default colors.
- [ ] View the banner with High Contrast mode on Windows. Go to the ‘Personalization’ settings in the Control Panel and view with custom colours.

**Comment by aeroplanejane** [comment]
Fixed on consent banners. Closing.

**Comment by HarryVerhoef** [comment]
This has been fixed on consent banners, but unsure if it has been fixed for the navigation banner

**Comment by greenc05** [comment]
Unsure why this was closed @aeroplanejane

**Comment by aeroplanejane** [comment]
@greenc05 sorry thought this applied to consent banners. Reopening for nav

**Comment by paruchurisilpa** [comment]
Checked in high contrast mode and LGTM..

http://psammead-preview.tools.bbc.co.uk/4475
